static/styles/subscriptions.css

```diff
@@ -800,6 +785,23 @@ h4.stream_setting_subsection_title {
 
         .stream-creation-body {
             padding: 15px;
+
+            section.block {
+                margin-bottom: 20px;
+            }
+
+            #announce-new-stream {
+                margin-top: 10px;
+
+                div[class^="fa"] {
+                    margin-left: 3px;
+                    margin-right: 8px;
+                }
+            }
+
+            input[type="radio"] {
+                margin: 3.5px 0 1px;
+            }
```

Comment:
For reference -> https://github.com/zulip/zulip/pull/21264/commits/5deca115197cb2d46d54ad31c290787a06d87080#r816249465